pmm-tests/pmm-framework.sh

```diff
@@ -706,6 +730,25 @@ setup(){
   echo -e "******************************************************************"
 }
 
+#Download and install PMM2.x client
+install_client(){
+ PMM_CLIENT_TAR_URL=$(lynx --listonly --dump https://www.percona.com/downloads/TESTING/pmm/ | grep  "pmm2-client" |awk '{print $2}'| grep "tar.gz" | head -n1)
+  echo "PMM2 client  $PMM_CLIENT_TAR_URL"
+  wget $PMM_CLIENT_TAR_URL
+  PMM_CLIENT_TAR=$(echo $PMM_CLIENT_TAR_URL | grep -o '[^/]*$')
+  tar -xzf $PMM_CLIENT_TAR
+  PMM_CLIENT_BASEDIR=$(ls -1td pmm2-client-* 2>/dev/null | grep -v ".tar" | head -n1)
+  pushd $PMM_CLIENT_BASEDIR > /dev/null
+  export PATH="$PWD/bin:$PATH"
+
+}
+
+configure_client() {
+  agent_id=$(pmm-admin register --server-url="http://"$IP_ADDRESS:$PMM_PORT  --server-insecure-tls|sed -n 's/.*agent_id//p') 
+ sudo pmm-agent  --insecure-tls --id="/agent_id$agent_id" --address=$IP_ADDRESS:443 > $PWD/pmm-agent.logs &
```

Comment:
`sudo` needed here?

Reply:
yes, it's known problem, going to be fixed